dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/translator/compiler/utils/JspFileManager.java

```diff
@@ -76,16 +76,23 @@ public JspFileManager(JavaFileManager fileManager, ClassLoader classLoader) {
         }
     }
 
+    // May be needed for Java Modules? Requires JDK 9+
+    // @Override
+    // public Iterable<Set<Location>> listLocationsForModules(Location location) throws IOException, UnsupportedOperationException, IllegalArgumentException {
+    //     return super.listLocationsForModules(location);
+    // }
+
     @Override
     public Iterable<JavaFileObject> list(Location location, String packageName, 
                                          Set<Kind> kinds, boolean recurse) throws IOException {
         if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled()&&logger.isLoggable(Level.FINE)) {
             logger.logp(Level.FINE, CLASS_NAME, "list", "Looking for classes in package = " + packageName + " from location = " + location.getName());
         }
         Iterable<JavaFileObject> results = new ArrayList<JavaFileObject>();
-        
         if (location == StandardLocation.PLATFORM_CLASS_PATH) // let standard manager handle
             results = super.list(location, packageName, kinds, recurse);
+        else if (JAVA_MAJOR_VERSION >= 9 && location.getName().equals("SYSTEM_MODULES[java.base]")) // base module for 9+
+            results = super.list(location, packageName, kinds, recurse);
```

Comment:
Due to Java 9 modules,  needed to add this snippet in to get the `useJDKCompiler` option working on Java 11+ 

Otherwise, the code would return and not find java.base causing a failure.